common/character_interactions/00_alliance.txt

```diff
@@ -97,6 +97,11 @@
 				}
```

Comment:
So because it's based on the abduction, the final event doesn't mention if the scheme was discovered unlike other schemes like murder.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117258236-cd085900-ae5d-11eb-86ce-f23271e04431.png)

</details>